Engine/Commands/InvokeScriptAnalyzerCommand.cs

```diff
@@ -416,36 +416,35 @@ protected override void EndProcessing()
             ScriptAnalyzer.Instance.CleanUp();
             base.EndProcessing();
 
-            var infoCount = diagnosticCounts[DiagnosticSeverity.Information];
-            var warningCount = diagnosticCounts[DiagnosticSeverity.Warning];
-            var errorCount = diagnosticCounts[DiagnosticSeverity.Error];
-            var parseErrorCount = diagnosticCounts[DiagnosticSeverity.ParseError];
+            var diagnosticCount = diagnosticCounts.Values.Sum();
 
             if (ReportSummary.IsPresent)
             {
-                var numberOfRuleViolations = infoCount + warningCount + errorCount;
-                if (numberOfRuleViolations == 0)
+                if (diagnosticCount == 0)
                 {
                     Host.UI.WriteLine("0 rule violations found.");
                 }
                 else
                 {
-                    var pluralS = numberOfRuleViolations > 1 ? "s" : string.Empty;
-                    var message = $"{numberOfRuleViolations} rule violation{pluralS} found.    Severity distribution:  {DiagnosticSeverity.Error} = {errorCount}, {DiagnosticSeverity.Warning} = {warningCount}, {DiagnosticSeverity.Information} = {infoCount}";
-                    if (warningCount + errorCount == 0)
-                    {
-                        ConsoleHostHelper.DisplayMessageUsingSystemProperties(Host, "WarningForegroundColor", "WarningBackgroundColor", message);
-                    }
-                    else
-                    {
-                        ConsoleHostHelper.DisplayMessageUsingSystemProperties(Host, "ErrorForegroundColor", "ErrorBackgroundColor", message);
-                    }
+                    var infoCount = diagnosticCounts[DiagnosticSeverity.Information];
+                    var warningCount = diagnosticCounts[DiagnosticSeverity.Warning];
+                    var errorCount = diagnosticCounts[DiagnosticSeverity.Error] + diagnosticCounts[DiagnosticSeverity.ParseError];
```

Comment:
Should parse errors be included in the error count, or displayed as separate parse error count? I really don't know. @bergmeister?